test/js/ez-platform-ui-app.js

```diff
@@ -150,6 +150,78 @@ describe('ez-platform-ui-app', function() {
         });
     });
 
+    describe('handle contentDiscover', function () {
+        const property = 'anything';
+        let eventFired = false;
+
+        function fireContentDiscover () {
+            element.dispatchEvent(new CustomEvent('ez:contentDiscover', {
+                detail: {
+                    config: {
+                        anyProp: property,
+                    },
+                    handlers: {
```

Comment:
actually here it's the struct of the detail attr I get from the event facade. To change this struct I need to alter some code out of this repo I guess...